Mage/src/main/java/mage/game/GameState.java

```diff
@@ -1229,4 +1231,12 @@ public void addPowerInsteadOfToughnessForDamageLethalityFilter(UUID source, Filt
                 .map(usePowerInsteadOfToughnessForDamageLethalityFilters::get)
                 .collect(Collectors.toList());
     }
+
+    boolean checkCommander(Card card, Game game) {
```

Comment:
Names are confused. I would rename to more understandable ones with the current zone ref. For example:
* `checkCommander()` -> `isCommanderKeepsInCurrentZone()`;
* `stuckCommanders` -> `commandersKeepsInCurrentZone`;
* `stickCommander` -> `keepCommanderInCurrentZone`;